pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -1159,6 +1159,80 @@ func TestImportControlJob(t *testing.T) {
 	})
 }
 
+// TestImportWorkerFailure tests that IMPORT can restart after the failure
+// of a worker node.
+func TestImportWorkerFailure(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	// TODO(mjibson): Although this test passes most of the time it still
+	// sometimes fails because not all kinds of failures caused by shutting a
+	// node down are detected and retried.
+	t.Skip()
```

Comment:
nit: i generally like to pass a brief explanation (or filed issue #) to Skip.

Reply:
fixed